CHANGELOG.rst

```diff
@@ -2,6 +2,180 @@
+  this API (they all default to false):
+  
+  - apm_config.remote_tagger
+  - logs_config.remote_tagger
+  - process_config.remote_tagger
+
+- New perf map usage metrics.
+
+- Add unofficial arm64 support to network tracer in system-probe.
+
+- system-probe: Add optional runtime compilation of eBPF programs.
+
+
+.. _Release Notes_7.26.0_Enhancement Notes:
+
+Enhancement Notes
+-----------------
+
+- APM: Sublayer metrics (trace.<SPAN_NAME>.duration and derivatives) computation
+  in agent can be disabled with feature flags disable_sublayer_spans, disable_sublayer_stats.
+  Reach out to support with questions about this metric.
+
+- APM: Automatically activate non-local trafic (i.e. listening on 0.0.0.0) for APM in containerized environment if no explicit setting is set (bind_host or apm_non_local_traffic)
+
+- APM: Add a tag allowing trace metrics from synthetic data to 
+  be aggregated independently.
+
+- Consider the task level resource limits if the container level resource limits aren't defined on ECS Fargate.
+
+- Use the default agent transport for host metadata calls.
+  This allows usage of the config ``no_proxy`` setting for host metadata calls.
+  By default cloud provider IPs are added to the transport's ``no_proxy`` list.
+  Added config flag ``use_proxy_for_cloud_metadata`` to disable this behavior. 
+
+- GOMAXPROCS is now set automatically to match the allocated CPU cgroup quota.
+  GOMAXPROCS can now also be manually specified and overridden in millicore units.
+  If no quota or GOMAXPROCS value is set it will default to the original behavior.
+
+- Added ``--flare`` flag to ``jmx (list|collect)`` commands to save check results to the agent logs directory.
+  This enables flare to pick up jmx command results.
+
+- Kubernetes events are now tagged with kube_service, kube_daemon_set, kube_job and kube_cronjob.
+  Note: Other object kinds are already supported (pod_name, kube_deployment, kube_replica_set).
+
+- Expose logs agent pipeline latency in the status page.
+
+- Individual DEB packages are now signed.
+
+- Docker container, when not running in a Kubernetes
+  environment may now be tailed from their log file.
+  The Agent must have read access to /var/lib/docker/containers
+  and Docker containers must use the JSON logging driver.
+  This new option can be activated using the new configuration
+  flag ``logs_config.docker_container_use_file``.
+
+- File tailing from a kubernetes pod annotation is
+  now supported. Note that the file path is relative
+  to the Agent and not the pod/container bearing
+  the annotation.
+
+
+.. _Release Notes_7.26.0_Known Issues:
+
+Known Issues
+------------
+
+- Remove Docker volumes for ``/etc/datadog-agent`` and ``/tmp`` as it prevents to inherit from Datadog Agent image. It was originally done to allow read-only rootfs on Kubernetes, so continue supporting this feature, relevant volumes are created in newer Kubernetes manifest or Helm chart >= 2.6.9
```

Comment:
(cc @vboulineau since you added this in https://github.com/DataDog/datadog-agent/pull/7222)

`Known issues` should only list known issues that are shipped in 7.26.0, which does not apply to this change here:
* if this change is an upgrade note that may be worth flagging for users on specific setups, we should move it to the `Upgrade notes`
* otherwise, if it's a misc update, it could be moved to the misc `Other notes` section

WDYT?

Also, general language update:

```suggestion
- Docker image: remove Docker volumes for ``/etc/datadog-agent`` and ``/tmp`` as it prevents to inherit from Datadog Agent image. It was originally done to allow read-only rootfs on Kubernetes, so in order to continue supporting this feature, relevant volumes are created in newer Kubernetes manifest or Helm chart >= 2.6.9
```

Reply:
I agree with @olivielpeau.
Even if should not most of the customer. It can be good to have this note in the "Upgrade notes" for user that build image on top of our image.

Reply:
OK thanks the your input. @ogaca-dd could you move this to the `Upgrade Notes` section? (no need to edit the original `releasenote` since we only use it once to generate the changelog)